app/src/main/java/com/kms/cura/model/UserModel.java

```diff
@@ -27,4 +41,41 @@ public boolean save(UserEntity user) {
         // call to api to save new user
         return false;
     }
+    public void registerPatient(final UserEntity Entity, final Context mContext){
+        StringRequest stringRequest = new StringRequest(Request.Method.POST, Settings.URL_CREATE_PATIENT_API, new Response.Listener<String>() {
```

Comment:
Should extract new Response.Listener<String>() to a new class for later re-use with other entity types